File: java-maven-spring-data-jdbc-dynamic/src/main/java/pl/exsio/querydsl/entityql/examples/spring_data_jdbc/entity/Group.java
package pl.exsio.querydsl.entityql.examples.spring_data_jdbc.entity;

import org.springframework.data.annotation.Id;

public class Group {

    @Id
    private Long id;
    private String name;
    private GroupAdmin admin;

    public Group(Long id, String name) {
        this.id = id;
        this.name = name;
    }

    @Override
    public String toString() {
        return "Group{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", admin=" + admin +
                '}';
    }
}

File: kotlin-gradle-spring-data-jdbc-dynamic/src/main/kotlin/pl/exsio/querydsl/entityql/examples/spring_data_jdbc/entity/Group.kt
package pl.exsio.querydsl.entityql.examples.spring_data_jdbc.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column

class Group(@Id @Column("GROUP_ID") var id: Long, var name: String, var admin: GroupAdmin?) {

    constructor(id: Long, name: String) : this(id, name, null)
}


